examples/ima.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <title>IMA examples</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+    <link href="//vjs.zencdn.net/5.3/video-js.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="http://gvabox.com/sbusolits/amp/css/videojs.ads.css" />
```

Comment:
Yeah this is not the best way to do this, since this will rely on the users copy-pasting the custom CSS your component need and paste it there.

A better way to load scripts/styles is to load them inside the 3p iframe. Take a look at [Twitter's integration and how we load their external script](https://github.com/ampproject/amphtml/blob/master/3p/twitter.js#L31). I think this also applies to external CSS loading.

@cramforce is this the right path to proceed with this?